pkg/chart/metadata.go

```diff
@@ -34,6 +34,9 @@ type Maintainer struct {
 
 // Validate checks valid data and sanitizes string characters.
 func (m *Maintainer) Validate() error {
+	if m == nil {
+		return ValidationError("the maintainer's item can not be nil")
```

Comment:
the maintainer's item can not be nil --> maintainer cannot be an empty list

Reply:
> the maintainer's item can not be nil --> maintainer cannot be an empty list

@hickeyma   this place checks maintainer, not list of maintainers.  " maintainer cannot be  empty"  may be better?

Reply:
Same comment applies as in https://github.com/helm/helm/pull/11044#discussion_r954867393